packages/@honkit/theme-default/src/less/website.less

```
@@ -1,5 +1,8 @@
 @import "base/all.less";
-@import "../../font-awesome/less/font-awesome.less";
+@import "../../../../../node_modules/@fortawesome/fontawesome-free/less/fontawesome.less";
```

Does `less` support module name resolution like the following?

```suggestion
@import "@fortawesome/fontawesome-free/less/fontawesome.less";
```